### PR TITLE
Create user entity

### LINKED ISSUE #3: Create DB User class
Create DB user class with the following fields

Image

### PR DESCRIPTION
**Pull request by lilygostovic**
closes #3

### CODE REVIEW

**Review by mkramarz — DISMISSED** [review]
Really good! Two small requests:
- Move the file into its own folder under `backend/src/entities/`
- Rename the file and class to "UserEntity"

File: backend/src/entities/UserEntity.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  username: string;
+
+  @Column()
+  email: string;
```
Comment (peterghrong) [inline backend/src/entities/UserEntity.ts:15]: Either email or Username might have to be unique?
Prob check with @kehwhy @cdjamalov

**Review by peterghrong — CHANGES_REQUESTED** [review]
Need Migration plan, code looks good aside from minor change
https://orkhan.gitbook.io/typeorm/docs/migrations

^ I take this back, Typeorm does migration automatically on startup I think

**Review by peterghrong — APPROVED** [review]
image

**Pull request merged**